docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative use case. You will learn how to create, send, and verify events in Knative.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](../install/README.md). 
+
+## Setting up Eventing subcomponents
+
+Before you start to send events, you need to create the components needed to transport the events. This guide walks you through creating each Eventing subcomponent.
+
+### Configuring namespace for eventing
+
+In most Knative tutorials, you will operate inside of a namespace. A namespace groups together subcomponents so that you can easily organize them.
+
+1. Create the namespace. This command creates a namespace called **event-example**:
+
+```sh
+kubectl create namespace event-example
+```
+
+2. Set up the namespace for Knative Eventing. To set up the namespace, add a label to your namespace with this command:
+
+```sh
+kubectl label namespace event-example knative-eventing-injection=enabled
+```
+
+The `knative-eventing-injection` label triggers Knative to add Knative Eventing subcomponents such as the `Broker` in your namespace.
+
+In the next section, you'll learn more about the `Broker`.
```

Comment:
This is true but I think whats more important is that you explain the relationship with what we have done, to whats next. For example, "Now that you have the `event-example` namespace, you can use it to create your Eventing subcomponents."